packages/lodestar/src/chain/validation/aggregateAndProof.ts

```diff
@@ -69,7 +69,12 @@ export async function validateGossipAggregateAndProof(
     });
   }
 
-  // TODO: check pool of aggregates if already seen (not a dos vector check)
+  if (await db.aggregateAndProof.hasAttestation(aggregate)) {
```

Comment:
Is this cache of seen aggregates in memory or in disk?

Reply:
it's in disc with key as aggregated attestation root

Reply:
If we get 600 aggregates per slot would that be "a lot" of disk reads? Would it be worth it to keep a the roots in memory in a set? Keep them only for 2 epochs and them drop them

Reply:
seems like SeenAttestationCache was not hit, will leverage it instead of disk read.